bin/propolis-server/src/lib/initializer.rs

```diff
@@ -186,7 +192,10 @@ impl<'a> MachineInitializer<'a> {
             );
             pci_builder.add_bridge(desc)?;
         }
-        let pci_topology = pci_builder.finish(self.inv, self.machine)?;
+        let pci::topology::FinishedTopology {
+            topology: pci_topology,
+            bridges: _bridges,
```

Comment:
The old `pci_builder` used to register the created bridges into the inventory. To match its behavior I think the bridges should be added to `devices` here instead of being dropped (they already have a `Lifecycle` impl).